.eslintrc.json

```diff
@@ -5,6 +5,11 @@
   "plugins": ["@typescript-eslint", "prettier"],
   "rules": {
     "no-console": 1, // Means warning
-    "prettier/prettier": 2 // Means error
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ]
   }
```

Comment:
I think this change is in another pr, right? Could you please remove it from this one?

Reply:
Will do. But I need it in until I can update this branch with the PR to fix this. SO we need to approve the other PR asap please.